src/OrchardCore.Modules/Orchard.Lucene/LuceneIndexingService.cs

```diff
@@ -50,80 +51,88 @@ public class LuceneIndexingService
 
         public async Task ProcessContentItemsAsync()
         {
-            
-            // TODO: Lock over the filesystem
+            // TODO: Lock over the filesystem in case two instances get a command to rebuild the index concurrently.
 
-            var allIndexes = new Dictionary<string, int>();
+            var allIndices = new Dictionary<string, int>();
 
             // Find the lowest task id to process
             int lastTaskId = int.MaxValue;
             foreach (var indexName in _indexManager.List())
             {
                 var taskId = _indexingState.GetLastTaskId(indexName);
                 lastTaskId = Math.Min(lastTaskId, taskId);
-                allIndexes.Add(indexName, taskId);
+                allIndices.Add(indexName, taskId);
             }
 
-            if (!allIndexes.Any())
+            if (!allIndices.Any())
```

Comment:
I am changing it, but FYI this doesn't change anything in the end. Internally this is an array, and all it costs internally (looked at the Linq source code) is a cast to `IList` and it then uses `Count` which is `Length` in the array implementation. So like a 1/1000 of a nano second ;) It's not even a perf sensitive code path.